research/model_description.ipynb

```diff
@@ -507,8 +516,175 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.scatter(time[0], derivative[0])\n",
```

Comment:
please, draw a dotted line and make it blue. Because all predictions on the graph below using this style